src/Garn/Edit.hs

```diff
@@ -0,0 +1,11 @@
+module Garn.Edit (editGarnTs) where
+
+import Cradle (StdoutTrimmed (..), run, run_)
+import Data.String.Conversions (cs)
+import Paths_garn (getDataFileName)
+
+editGarnTs :: [String] -> IO ()
+editGarnTs args = do
+  editorNix <- getDataFileName "nix/editor.nix"
+  StdoutTrimmed path <- run "nix-build" [cs editorNix, "--no-out-link"]
+  run_ (cs path <> "/bin/vscodium") ("garn.ts" : args)
```

Comment:
I think in the future we probably want to implement something like `garn edit --vim-style-vscodium` or `garn edit --vim` or `garn edit --emacs`. Is that just going to be a breaking change then? What's the use-case for allowing to pass arguments to `vscodium`?

Reply:
The main use-case is writing tests! But currently you still need to pass `--`, so it would still be compatible with `--vim` or whatever. (I have a hard time knowing what 'breaking change' in this context means though, and whether it matters.)

We could have e.g. an environment variable instead, that corresponds to the arguments passed. But that feels quite hacky too (and also `cradle` doesn't support env variables yet.) Open to other suggestions though!

Reply:
Oh, right, you have to use `--`. That's a bit surprising, but I think fine.